### PR TITLE
updates for 1.0.2 with participant processor additions

### PR DESCRIPTION
**Pull request by danaskallman**
Some tweaks to docs for latest in master

### CODE REVIEW

File: docs/processors.md [hunk 1]
```diff
@@ -99,22 +99,29 @@ The CiviCRM Contribution provides a way to use a Caldera Forms calculation field
...
 ### Memberships
 
-The CiviCRM Membership provides the ability to register and renew memberships. By using this processor you can renew existing memberships as is done using a contribution page, or create new memberships at each renewal. 
+The CiviCRM Membership processor provides the ability to register and renew memberships. By using this processor you can renew existing memberships as is done using a contribution page, or create new memberships at each renewal. If multiple memerships are provided, use one processor per membership. For free memberships the processor can work on it's own, for paid events use alongside the Line Item and Order processors. 
 
 ![CFC Membership Processor](./images/cfc-membership-processor.png)
 
+### Event Participants
+
+The CiviCRm Participant processor provides the ability to add event registration, both free and paid, to a form. If multiple registrations are provided, use one processor per event. For free events the processor can work on it's own, for paid events use alongside the Line Item and Order processors.
```
Comment (kcristiano) [inline docs/processors.md]: Typo CiviCRm/CiviCRM

File: docs/custom-fields.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ When adding new fields to your Caldera form there are two new options available
 * CiviCRM Country
 * CiviCRM State/Provice
 * Contact Reference
+* CiviCRM Discount (with CiviDiscount installed)
```
Comment (kcristiano) [inline docs/custom-fields.md]: would Requires CiviDiscount Extension be a better description?

File: docs/processors.md [hunk 3]
```diff
@@ -99,22 +99,29 @@ The CiviCRM Contribution provides a way to use a Caldera Forms calculation field
 
 ### Memberships
 
-The CiviCRM Membership provides the ability to register and renew memberships. By using this processor you can renew existing memberships as is done using a contribution page, or create new memberships at each renewal. 
+The CiviCRM Membership processor provides the ability to register and renew memberships. By using this processor you can renew existing memberships as is done using a contribution page, or create new memberships at each renewal. If multiple memerships are provided, use one processor per membership. For free memberships the processor can work on it's own, for paid events use alongside the Line Item and Order processors.
```
Comment (kcristiano) [inline docs/processors.md]: Not sure what `, for paid events use alongside the Line Item and Order processors. ` is trying to explain.   Do we want to say that the Membership Processor can be used along with the Event Processor as long as you make use of Line Items and the Order processor?

File: docs/processors.md [hunk 4]
```diff
@@ -99,22 +99,29 @@ The CiviCRM Contribution provides a way to use a Caldera Forms calculation field
...
+
+![CFC Participant Processor](./images/cfc-participant-processor.png)
 
 ### Price Set Line Items
 
-The CiviCRM Line Item connects Caldera form fields to Line Items in a price set to record transaction details. When using the CiviCRM Membership Processor you can connect the contributions details to the membership record. Once the membership processor is setup a unique variable ID is created and available for selection in the Line Item processor to connect these two entities. 
+The CiviCRM Line Item connects Caldera form fields to Line Items in a price set to record transaction details. When using the CiviCRM Membership Processor or the CiviCRM Event Participant processor you can connect the contributions details to the membership or participant record. Once the membership or participant processor is setup a unique variable ID is created and available for selection in the Line Item processor to connect these two entities.
```
Comment (kcristiano) [inline docs/processors.md]: `or participant`  s/b `or Participant` to be consistent

File: docs/processors.md [hunk 5]
```diff
@@ -99,22 +99,29 @@ The CiviCRM Contribution provides a way to use a Caldera Forms calculation field
...
 ![CiviCRM Line Item with Memberhips Processor](./images/civicrm-line-item-with-membership-processor.jpg)
 
+![CiviCRM Line Item with Participant Processor](./images/cfc-participant-line-item.png)
+
 ![CiviCRM Line Item Donation Processor](./images/civicrm-line-item-donation-processor.jpg)
 
 ### Orders
 
-The CiviCRM Order processor is where you can connect everything into one contribution record, by mapping all line items into one order.
+The CiviCRM Order processor is where you can connect everything into one contribution record, by mapping all line items into one order that will be recorded on in the Contribution entity.
```
Comment (kcristiano) [inline docs/processors.md]: Not sure of 'everything'.  Do we want to say "The CiviCRM Order processor is where you will connect    all other processor to one contribution record.  This is done by mapping all line items to one order that will be recorded  in the Contribution entity"

File: docs/processors.md (hunk 3 above)
Comment (danaskallman) [reply docs/processors.md]: This means that each participant or membership processor needs a line item if it's a paid. So say if there were 2 event processors and 1 membership processors means 3 line item processors to then map into the order processor. If that makes any sense.

File: docs/processors.md (hunk 5 above)
Comment (danaskallman) [reply docs/processors.md]: Hmmm, this is for adding all line items into the order processor. I'll see how to say this better.

**Comment by danaskallman** [comment]
Made tweaks, let me know they make sense.

File: docs/processors.md [hunk 6]
```diff
@@ -99,22 +99,29 @@ The CiviCRM Contribution provides a way to use a Caldera Forms calculation field
...
 ![CiviCRM Line Item with Memberhips Processor](./images/civicrm-line-item-with-membership-processor.jpg)
 
+![CiviCRM Line Item with Participant Processor](./images/cfc-participant-line-item.png)
+
 ![CiviCRM Line Item Donation Processor](./images/civicrm-line-item-donation-processor.jpg)
 
 ### Orders
 
-The CiviCRM Order processor is where you can connect everything into one contribution record, by mapping all line items into one order.
+The CiviCRM Order processor is where you each Line Item Processor is connected into one order that then contribution record.
```
Comment (kcristiano) [inline docs/processors.md]: @danaskallman I think this got mangled in your edits.    

```The CiviCRM Order processor is where each Line Item Processor is connected into one order that then creates the contribution record.``` 

Is that better ^

**Comment by mecachisenros** [comment]
Thank you for this @danaskallman and @kcristiano!

**Pull request merged**